model/task/db.go

```diff
@@ -594,6 +595,13 @@ func GetRecentTasks(period time.Duration) ([]Task, error) {
 		return nil, errors.Wrap(err, "problem with stats query")
 	}
 
+	for i, task := range out {
+		if err := task.MergeNewTestResults(); err != nil {
+			return nil, errors.Wrap(err, "error merging new test results")
+		}
+		out[i] = task
```

Comment:
might out be nil rather than empty? (does that matter?)

Reply:
We actually don't need this, since test result isn't used for anything by the caller.